packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -129,6 +134,7 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
      * @param error - optional error object to log
      */
     public sendTelemetryEvent(event: ITelemetryGenericEvent, error?: any) {
+        stringifyEventFields(event);
```

Comment:
Looks redundant to me

Reply:
And elsewhere.  You should only need to call this from within `sendTelemetryEventCore`